### ISSUE TITLE
GPU and AD compatibility

### ISSUE THREAD

**Issue by AzamatB** [Jutho/Strided.jl#7]
It would be nice to make `@strided` macro work with `CuArrays.jl` and `Zygote.jl`

**Comment by Jutho** [comment]
I fully agree. However, I don't have the knowhow to do either of those at this point. Hopefully in the future.

**Comment by dynamic-queries** [comment]
Hello, 
I am willing to invest time to contribute to such an implementation. 
Do you have anything in mind ?

**Comment by lkdvos** [comment]
The future is now! 🥳  (at least for the GPU part)
Should be greatly improved once #51 is officially released.
I will close this issue anyways since it is somewhat outdated.
